### DISCUSSION TITLE
Special characters (czech)

### DISCUSSION THREAD

**Discussion by krystofstys** [QuestPDF/QuestPDF#901] [Q&A]
Hello,

I am trying to migrate my current solution using DinkToPdf to QuestPDF, but I face difficulties displaying Czech characters.

So far, I have tried using the basic fonts, which should support Czech characters, such as Arial or Times New Roman, to no avail.

I additionally tried adding a font which should have Czech character support:
```
QuestPDF.Settings.License = QuestPDF.Infrastructure.LicenseType.Community;

FontManager.RegisterFontWithCustomName("Atomic Age", File.OpenRead("Fonts/atomicage-regular.ttf"));

await Document.Create(container =>
{
    container.Page(page =>
    {
        page.Size(PageSizes.A4);
        page.Margin(2, QuestPDF.Infrastructure.Unit.Centimetre);

        page.Header().Text("Test dovedností mladých hasičů").SemiBold().FontSize(30).FontFamily("Atomic Age");
        });
}).ShowInPreviewerAsync();
```

I was facing similar issues in the other library, but changing the culture fixed the issue, which did not help in this case:
`Thread.CurrentThread.CurrentUICulture = CultureInfo.GetCultureInfo("cs");`

![image](https://github.com/QuestPDF/QuestPDF/assets/83222554/03f8fcc4-0673-4fc4-90a2-5ce61a07cb79)

I have the language installed in Windows.

I will be grateful for any suggestions.

**Comment by MarcinZiabek** [comment]
What version of the library are you using? I have just tested it with `2024.3.7` version, using a default `Lato` font, and everything looks as expected.

**Reply by krystofstys** [reply]
I have also used the `2024.3.7` version. I saw your notification and woke my notebook from sleep, and the issue went away after `hot reload` from Visual Studio.

I'm unsure why it went away, as I previously tried restarting the app and reloading. The only change was that I started the app via the debugger in Visual Studio, whereas previously, I followed Nick Chapsas’ video, which he called `dotnet watch`. Now, even calling `dotnet watch` creates a correct PDF.

Update: It seems to be an issue connected to running the `dotnet watch` command in the package manager console. And maybe the settings of the package manager console, as the special characters inside it, are also displayed incorrectly.

Thank you for your time.

**Comment by DawidRobertNowak** [comment]
Hello,

We are closing this thread due to inactivity. We apologize for the delayed response. Thank you for taking the time to reach out and contribute to the project — your feedback helps us improve QuestPDF.

Should you encounter any further problems, please don't hesitate to open a new issue.

Thank you!
